openmetadata-service/src/main/java/org/openmetadata/service/search/elasticsearch/ElasticSearchClient.java

```diff
@@ -1297,6 +1297,23 @@ private static SearchSourceBuilder buildSearchAcrossIndexesBuilder(
         buildSearchQueryBuilder(query, SearchIndex.getAllFields());
     FunctionScoreQueryBuilder queryBuilder = boostScore(queryStringBuilder);
     SearchSourceBuilder searchSourceBuilder = searchBuilder(queryBuilder, null, from, size);
+    searchSourceBuilder.aggregation(
+        AggregationBuilders.terms("database.name.keyword")
+            .field("database.name.keyword")
+            .size(MAX_AGGREGATE_SIZE));
+    searchSourceBuilder.aggregation(
+        AggregationBuilders.terms("databaseSchema.name.keyword")
+            .field("databaseSchema.name.keyword")
+            .size(MAX_AGGREGATE_SIZE));
+    return addAggregation(searchSourceBuilder);
```

Comment:
Should we do the aggregation on the fullyQualifiedName? Using name I believe if we have `service.database.schema` and `otherService.database.schema` then they'll both get aggregated under the same if we use `name` I believe.

Reply:
hi @TeddyCr thanks for pointing your concern is valid, but for our usage we are not  dependent
only on a single aggregation while getting result, we check the condition with other aggregations as well
<img width="1106" alt="image" src="https://github.com/user-attachments/assets/fe7347df-92f8-424e-9a85-427e15a6ec24">

Reply:
have tried to replicate the same scenario with same database and schema, seems to be working fine.
Also will be removing the support for aggregations on name.keyword, as in most scenario we are moving to show results based on displayName

<img width="506" alt="image" src="https://github.com/user-attachments/assets/34859de9-6344-4799-a9f9-cf6152adfc3c">